### PR TITLE
Health authorities

### PR DESCRIPTION
**Pull request by adnanjt**
Screenshot_20200603-155553
Screenshot_20200603-155538

https://trello.com/c/0WZekEYC/7-as-a-positive-patients-i-want-my-path-history-so-that-i-can-keep-people-around-me-safe

#### Description:

As a positive patients, I want my path history so that I can keep people around me safe

#### How to test:
This is mostly backend changes that are not going to be testable. 

Exposed API it's working but you need to ask for critical points or change on request to have a match. 

Also on first entry to LocationTracking Screen a new pop notification appears about health authorities and you can check your health authorities on settings

### CODE REVIEW

File: app/services/HCAService.js [hunk 1]
```diff
@@ -6,11 +6,14 @@ import { AUTHORITIES_LIST_URL } from '../constants/authorities';
 import {
   AUTHORITY_SOURCE_SETTINGS,
   ENABLE_HCA_AUTO_SUBSCRIPTION,
+  LOCATION_DATA
 } from '../constants/storage';
 import { GetStoreData, SetStoreData } from '../helpers/General';
 import languages from '../locales/languages';
 import { LocationData } from './LocationService';
 
+//const authoritiesYaml = require('./../constants/DR/healthcare-authorities.yaml');
```
Comment (RutsSantos) [inline app/services/HCAService.js]: Is this comment necessary?

**Review by RutsSantos — APPROVED** [review]
LGTM, 
Left a comment

File: app/services/HCAService.js (hunk 1 above)
Comment (adnanjt) [reply app/services/HCAService.js]: not really will take it out thanks

File: app/services/HCAService.js [hunk 2]
```diff
@@ -93,28 +91,30 @@ class HCAService {
...
     }
+    this.offsetLocation(mostNorthEastPoint);
+    this.offsetLocation(mostSouthWestPoint);
 
     const baseUrl = 'https://webapps.mepyd.gob.do/contact_tracing/api/Contact';
     const url = mostNorthEastPoint && mostSouthWestPoint ? `${baseUrl}?NE=${mostNorthEastPoint.latitude},${mostNorthEastPoint.longitude}&SW=${mostSouthWestPoint.latitude},${mostSouthWestPoint.longitude}` 
                                       : baseUrl;
-    console.log("URL: "+ url);
-    console.log(" are there points? "+ mostNorthEastPoint && mostSouthWestPoint );
+    console.log("URL: "+ baseUrl);
```
Comment (inteljudajocu) [inline app/services/HCAService.js]: Remove this console log

**Review by inteljudajocu — APPROVED** [review]
LGTM just address the comments from @RutsSantos and this comment

**Pull request merged**